### PR TITLE
fix(nuxi): support `--inspect` in dev mode

### LINKED ISSUE #21312: Running Nuxt 3 development server with debugging mode not available?
### Environment

------------------------------
- Operating System: Darwin
- Node Version:     v18.16.0
- Nuxt Version:     3.5.0
- Nitro Version:    2.4.1
- Package Manager:  yarn@1.22.10
- Builder:          vite
- User Config:      srcDir, app, alias, css, modules, plugins, imports, logLevel, vite
- Runtime Modules:  ~/modules/eventHandler/index.js, ~/modules/eventBus/index.js, ~/modules/router/index.js, @pinia/nuxt@0.4.10
- Build Modules:    -
------------------------------

### Reproduction

Run below command:

```bash
node --inspect node_modules/.bin/nuxt dev
```

### Describe the bug

I want to start Nuxt development server while enabling the debug mode. For usual case if I run a node server with debug flag, I get a prompt message something like below:

```
Debugger listening on ws://127.0.0.1:9229/081fbb8d-b3e0-4ef5-92e6-54242327a280
```

And also by that point I can find the debugging tool for the server is activated by looking at the chrome inspector tool:

debug

But if I run that command with Nuxt 3 (3.5.0), it runs with same config as I run just `nuxt dev`. This did work with Nuxt 2 though, is there a change with how to run Nuxt server in debugging mode?

### Additional context

_No response_

### Logs

_No response_

### PR DESCRIPTION
**Pull request by Seboran**
### 🔗 Linked issue

resolves #21312

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)
- [x] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Adds the inspector option if present.

I don't know a lot about inspector options but this bug affects me so I gave it a try.

I mostly took inspiration from `packages/nuxi/src/cli.ts`

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: packages/nuxi/src/cli-wrapper.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@
 import { fileURLToPath } from 'node:url'
 import { fork } from 'node:child_process'
 import type { ChildProcess } from 'node:child_process'
+import mri from 'mri'
```
Comment (danielroe) [inline packages/nuxi/src/cli-wrapper.ts:7]: Can we do this without importing `mri` in this file? I think the parsing is simple enough for `--inspect`.

**Review by pi0 — APPROVED** [review]
Nice! LGTM as an experimental feature. We need to rework this for next gen cli ~> https://github.com/nuxt/cli/issues/73

**Comment by danielroe** [comment]
Thank you ❤️

**Comment by carbdias** [comment]
How can we use this? Does it console.log things to navigator inspector ? Or inside Visual Studio Code?

**Pull request merged**